src/OpenLoco/src/Ui/Window.h

```diff
@@ -215,7 +215,11 @@ namespace OpenLoco::Ui
         uint16_t var_83C;
         uint16_t rowHeight;
         int16_t rowHover = -1;
-        int16_t var_842;
+        union
+        {
+            int16_t orderTableIndex;
```

Comment:
just thinking - should these be defaulted to -1?

Reply:
Seems reasonable to do, I think in unions you can only default one member but in this case both are same type so that doesn't matter.

Reply:
done